tests/example.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+from argparse_dataclass import dataclass
+
+
+@dataclass
+class Opt:
+    x: int = 42
+    y: bool = False
+
+
+def main():
+    params = Opt.parse_args()
+    print(params)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
For consistency I would prefer if this example goes in the main docstring to be tested with `doctest`.

Reply:
I added the docstring as well as a standalone tests/example.py so people can easily play with the example.